littlebird/tweet_tokenizer.py

```diff
@@ -123,28 +122,36 @@ def tokenize_tweet_file(self, input_file, sample_size=-1, return_tokens=False):
         # Check for empty file
         if num_tweets == 0:
             logging.warning(f"{input_file} has no tweets.")
-            return
+            return None
 
         # Sample from the file's tweets
         if sample_size != -1:
             if sample_size < num_tweets:
                 all_tweet_text = random.sample(all_tweet_text, k=sample_size)
-        
+
         # Tokenize the tweets and return
         # Some tweets have no valid tokens. Skip them.
-        tweet_text = map(self.tokenize, all_tweet_text)
+        tweet_text_ = map(self.tokenize, all_tweet_text)
+        tweet_text: Union[List[str], List[List[str]]]
         if return_tokens:
-            tweet_text = [t for t in tweet_text if t != []]
+            tweet_text = [t for t in tweet_text_ if t != []]
```

Comment:
Intermediate variable?

Reply:
So the type of each label doesn't change. Suppose it's not necessary in hindsight